src/dsp_tools/commands/excel2json/new_lists.py

```diff
@@ -352,73 +365,69 @@ def _get_preferred_language(columns: pd.Index[str], ending: str = r"(\d+|list)")
     raise InputError(msg)
 
 
-def _make_all_formal_excel_compliance_checks(
-    excel_dfs: dict[str, dict[str, pd.DataFrame]],
-) -> None:
+def _make_all_formal_excel_compliance_checks(list_files: list[ExcelFile]) -> None:
     """
     Check if the excel files are compliant with the expected format.
 
     Args:
-        excel_dfs: dictionary with the excel file name as key
-                    and a dictionary with the sheet name as key and the dataframe.
+        list_files: Class with the filename and the sheets
```

Comment:
That's not very precise. Firstly, it is not a class, but rather an instance/object. Secondly, it's a list of such objects, not a single object.
Since darglint allows you to omit the "Args", you could consider omitting it.